build/media-suite_compile.sh

```diff
@@ -173,8 +173,8 @@ if [[ $ripgrep = y || $rav1e = y || $dssim = y ]]; then
 
     _check=(bin/sccache.exe)
     if do_vcs "https://github.com/mozilla/sccache.git"; then
-        do_uninstall "${_check[@]}"
         do_rust
+        sccache --stop-server >/dev/null 2>&1 || false
```

Comment:
Shouldn't we check if sccache exists first?

Reply:
```suggestion
        sccache --stop-server >/dev/null 2>&1 || true
```

That's what the to dev nulll and 2>&1 are for, just ignore if it fails

But I did make the mistake of putting false there